exchange/exchangetest/append-bidder-names.json

```diff
@@ -166,9 +166,6 @@
     },
     "ext": {
       "debug": {
-        "httpcalls": {
```

Comment:
Do we still have test coverage somewhere that `httpcalls` are included if debug is allowed?

Reply:
Yes, exchange_test.go, `TestDebugBehaviour` covers all the cases